core/arch/arm/plat-rpi3/conf.mk

```diff
@@ -0,0 +1,70 @@
+ #
```

Comment:
Personally I don't mind, but since it seems to be common practice not only in OP-TEE to not have license information in makefiles etc, I'll remove it.